### PR TITLE
changed all avatars to initials + colors

### PR DESCRIPTION
**Pull request by NEUDitao**
they're beautiful I think

### CODE REVIEW

**Review by willstenzel — APPROVED** [review]
Good stuff!

Only thing is we might want to change the font on the large avatars (like the ones on the today page) to have the same text to size ratio as the profile. @liustanley might have an opinion on that

image

**Review by liustanley — APPROVED** [review]
big brain looks good

i agree with will, just fix the text ratio on the today page and it's good to merge

**Review by liustanley — CHANGES_REQUESTED** [review]
wait nvm lol some changes that need to be made:
- [ ] Make standard text to avatar size ratio, consistency here is important
- [ ] Missed the avatar on `TAQueueCard`

**Review by liustanley — APPROVED** [review]

File: packages/app/components/Queue/TA/TABanner.tsx [hunk 1]
```diff
@@ -48,11 +50,20 @@ export default function TABanner({
...
-              icon={<UserOutlined />}
-              src={helpingQuestion.creator.photoURL}
-            />
+              style={{
+                backgroundColor: nameToRGB(helpingQuestion.creator.name),
+                fontSize: 36,
+              }}
+            >
+              {getInitialsFromName(helpingQuestion.creator.name)}
+            </Avatar>
```
Comment (dajinchu) [inline packages/app/components/Queue/TA/TABanner.tsx:66]: Very similar Avatar code appears 7 times. Is it possible to abstract?

**Review by dajinchu — APPROVED** [review]

**Pull request merged**